### PR TITLE
svg: code refactoring

### PR DESCRIPTION
**Pull request by hermet**
-- compiler warnings on msvc

### CODE REVIEW

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 1]
```diff
@@ -647,21 +647,9 @@ static bool _hslToRgb(float hue, float satuation, float brightness, uint8_t* red
...
-    _green = (f <= 0.5) ? i : (i + 1);
-
-    i = static_cast<uint8_t>(_blue * 255.0);
-    f = (_blue * 255.0) - i;
-    _blue = (f <= 0.5) ? i : (i + 1);
-
-    *red = static_cast<uint8_t>(_red);
-    *green = static_cast<uint8_t>(_green);
-    *blue = static_cast<uint8_t>(_blue);
+    *red = static_cast<uint8_t>(ceil(_red * 255.0f));
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgLoader.cpp:650]: why ceil()? roundf() seems to be correct.

Comment (hermet) [reply src/loaders/svg/tvgSvgLoader.cpp]: by mistake. thanks.

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 2]
```diff
@@ -726,7 +714,7 @@ static bool _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
         content = str + 4;
         content = _skipSpace(content, nullptr);
         if (_parseNumber(&content, &hue, &th) && hue) {
-            th = static_cast<uint32_t>(th) % 360;
+            th = float(uint32_t(th) % 360);
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgLoader.cpp:717]: There seems to be a problem with the existing source, How about using fmod()?

Comment (hermet) [reply src/loaders/svg/tvgSvgLoader.cpp:717]: I think you can fix it properly. This pr is just about a code refactoring due to the compiler warnings.

**Review by JSUYA — APPROVED** [review]

**Pull request merged**